devel/ansible/roles/dev/files/.bashrc

```diff
@@ -0,0 +1,6 @@
+# .bashrc
+
+alias fas2ipa-resetdata="sudo ipa-restore /var/lib/ipa/backup/`sudo ls /var/lib/ipa/backup` -p adminPassw0rd!"
```

Comment:
This will fail if there is more than one backup, you can use `ls -1rt /var/lib/ipa/backup | tail -1` in the subshell to avoid that.